ark/utils/segmentation_utils.py

```diff
@@ -71,8 +71,8 @@ def split_large_nuclei(cell_segmentation_mask, nuc_segmentation_mask, cell_ids):
 
             nuc_mask = nuc_segmentation_mask == nuc_id
 
-            # only proceed if parts of the nucleus are outside of the cell
-            if nuc_count != np.sum(nuc_mask):
+            # only proceed if a non-negligible part of the nucleus is outside of the cell
+            if np.sum(nuc_mask) - nuc_count > 5:
```

Comment:
this 5 should probably be a `min_size` parameter in the `split_large_nuclei` function.